### PR TITLE
refactor: move cadnode

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**

### CODE REVIEW

File: viewer/packages/rendering/src/rendering/EffectRenderManager.ts [hunk 1]
```diff
@@ -311,107 +286,115 @@ export class EffectRenderManager {
...
+      this._sceneComponentsProvider.setVisibilityOfSimpleSectors(true);
     }
   }
 
+  private preRenderSetup(): [
+    {
+      autoClear: boolean;
+      clearAlpha: number;
+      renderMode: RenderMode;
+    },
```
Comment (christjt) [inline viewer/packages/rendering/src/rendering/EffectRenderManager.ts:319]: This probably deserves a type

File: viewer/packages/rendering/src/sector/SceneComponentsProvider.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  prepareNormalScene(): void;
+  getNormalScene(): THREE.Scene;
+  restoreNormalScene(): void;
+
+  prepareInFrontScene(): void;
+  getInFrontScene(): THREE.Scene;
+  restoreInFrontScene(): void;
+
+  getGhostScene(): THREE.Scene;
```
Comment (christjt) [inline viewer/packages/rendering/src/sector/SceneComponentsProvider.ts:23]: Any way to reduce this into only two functions? With only `splitScene()` and `restoreScene()`?

**Review by christjt — CHANGES_REQUESTED** [review]

**Comment by larsmoa** [comment]
Dead, @haakonflatval-cognite ?

**Comment by christjt** [comment]
@larsmoa @haakonflatval-cognite Some of the changes here do not make sense anymore after the rendering pipeline changes. I will reimplement what has been done here. So we can leave it up until that is done (might take a lot of inspiration from this PR).

**Comment by christjt** [comment]
Closing this in favor of: https://github.com/cognitedata/reveal/pull/2173

**Pull request closed without merging**